src/vs/workbench/contrib/terminal/test/node/terminalProfiles.test.ts

```diff
@@ -35,33 +32,35 @@ suite('Workbench - TerminalProfiles', () => {
 				// 	assert.deepStrictEqual(expected, profiles);
 				// });
 				test('should detect Git Bash and provide login args', async () => {
-					const _paths = [`C:\\Program Files\\Git\\bin\\bash.exe`];
-					let config = {
-						profiles: {
-							windows: {
-								'Git Bash': {
-									source: 'Git Bash'
-								},
-							},
-						},
-						detectWslProfiles: false
-					};
-					const profiles = await detectAvailableProfiles(true, undefined, config, undefined, undefined, createStatProvider(_paths));
-					const expected = [{ profileName: 'Git Bash', path: _paths[0], args: ['--login'] }];
-					assert.deepStrictEqual(profiles, expected);
+					// const _paths = [`C:\\Program Files\\Git\\bin\\bash.exe`];
```

Comment:
Is this being fixed up later? Need an issue?

Reply:
there's an issue already